packages/tx/src/index.ts

```diff
@@ -1,5 +1,6 @@
 export { FeeMarketEIP1559Transaction } from './eip1559Transaction'
 export { AccessListEIP2930Transaction } from './eip2930Transaction'
+export { BlobEIP4844Transaction } from './eip4844Transaction'
```

Comment:
Fine by me.  How do you want to name it - `ShardBlobEIP4844Transaction`?

Reply:
Yes, I would prefer that if you're ok here.

Reply:
Sounds good. Will do a global find/replace.

Reply:
well its not Sharding yet since the blocks are not actually being sharded and thats why the CL language just refer then entire thing as eip4844. to me. I guess current name is good enough as Sharding will imply something different in future

Reply:
The EIP is called "Shard Blob Transactions" and I still think that people can/could associate this better if we have the "Shard" part in the name, since many will likely not know what a "Blob" is and they might also not be able to associate the EIP number 4844 (though this one will get pretty prominent to admit, similar to 1559). Anyhow, again, not so strong opinion here any more, interesting though that CL names this differently respectively with some shifted emphasis.